symfony/src/Controller/DefaultController.php

```diff
@@ -2,29 +2,153 @@
 
 namespace App\Controller;
 
+use App\Entity\User;
+use App\Service\AuthService;
 use App\Service\LoggerService;
+use App\Service\SiteService;
+use Pmi\Audit\Log;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Symfony\Component\Routing\Annotation\Route;
-use Pmi\Audit\Log;
+use Symfony\Component\Security\Csrf\CsrfToken;
 
-/**
- * @Route("/s")
- */
```

Comment:
Can we leave this prefix here, and then have the index route continue to live at `/`, which turns into `/s/`? That way we only have one place to update when we get rid of the `/s/` prefix. Also, I think `/s/` makes more sense anyways.

Reply:
(And if you do make that change, be sure to update the redirect URL, too)